distribution/openrct2.d.ts

```diff
@@ -1212,8 +1214,7 @@ declare global {
         readonly endY: number;
 
         /**
-         * The relative ending Z position. Negative numbers indicate
-         * that the track ends upside down.
+         * The relative ending Z position from the base of the first track sequence block.
```

Comment:
Negative numbers indicate the starting position is below the height of the first track sequence block. CableLiftHill, FlatToDown60LongBase, Down60ToFlatLongBase have negative endZ values.